package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Blue Ocean acceptance tests",
   "main": "index.js",
   "scripts": {
-    "test": "nightwatch"
+    "test": "npm run nightwatch",
+    "nightwatch": "PATH=./node:./node/npm:$PATH nightwatch",
```

Comment:
Oh actually @scherler .... no need to add it since it's already in there automatically when running `npm run`.